controllers/postController.js

```diff
@@ -86,6 +86,20 @@ router.put('/:id', verifyToken, (req, res) => {
   });
 });
 
+/**
+ * Like a post
+ */
+router.post('/:pid/like', verifyToken, (req, res) => {
+  Post.likePost(req.params.pid, req.user._id, req.body.isLiked)
```

Comment:
Any way we could rename `isLiked` to `addLike`? To me `isLiked` suggests we're flagging whether this user has already liked the post (how we interpret it client-side at the moment), when in fact what we're really asking is "should we add or take away a like?". Lmk if I've misunderstood the code though :stuck_out_tongue_closed_eyes: